README.md

```diff
@@ -1 +1,167 @@
-# Hello_Microverse_Project
\ No newline at end of file
+<a name="My Portfolio"></a>
+
+<div align="center">
+  <img src="Albert" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>My Portfolio</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [My Portfolio] <a name="about-project"></a>
+
+> This is my portfolio I build it with html and css.
+
+## 🛠 Built With <a name="built-with"></a>
+1. html
+2. css
+3. Linters
+
+### Tech Stack <a name="tech-stack"></a>
+1. html
+2. css
+3. Linters
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://indexhtml.org/">HTML</a></li>
+    <li><a href="https://stylecss/">CSS</a></li>
+    <li><a href="https://linters.org/">Linters</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+Added index.html file.
+Added style.css file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+Live-demo will be added soon.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Web browser
+Code editor
+
+
+### Setup
+
+Clone this repository to your desired folder:
+Run this command:
+https://github.com/albertkantwi/MyPortfolio
+
+### Install
+
+Install this project with:
+
+  cd my-project
+  npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+### Run tests
+
+To run tests, run the following command:
+
+Coming soon.
+
+### Deployment
+
+Coming soon
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+Albert Antwi
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@albertkantwi](https://github.com/albertkantwi)
+- Twitter: [@Albert_KAntwi](https://twitter.com/Albert_KAntwi?t=cdjVl5R1Vagv3xGkhypQmA&s=08)
+- LinkedIn: [Albert Antwi](https://www.linkedin.com/in/albert-antwi-557291263)
```

Comment:
- [ ] This link is broken. kindly consider placing a working link to make your readme professional.